cloudomate/hoster/vps/vps_hoster.py

```diff
@@ -0,0 +1,99 @@
+from cloudomate import wallet as wallet_util
+
+from cloudomate.hoster.hoster import Hoster
+
+
+class VpsHoster(Hoster):
```

Comment:
Since this is a very important class in Cloudomate, please add a few lines of comments here.